config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml

```diff
@@ -372,3 +372,40 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-conformance-k8s-ci-v1beta1
+  - name: pull-cluster-api-provider-azure-e2e-workload-upgrade-v1beta1
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 4h
+    branches:
+    - ^release-1.*
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220726-61b7bafca6-1.24
```

Comment:
this one got lost in the queue, can you please rebase and bump the image here if we still want to merge it?

Reply:
The purpose of this addition was to avoid breaking the periodc jobs. Because we run so few tests against the release branch(es), should we just set this to required to ensure we always get coverage before a merge?

Reply:
@CecileRobertMichon in case this got lost

Reply:
Good idea. This PR will give us two opportunities to smoke test once it lands and cherry-pick PRs are created:

https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2767

Reply:
we still need to bump the image to merge

Reply:
thanks, done!